platform/libs.flatlaf/nbproject/project.xml

```diff
@@ -28,6 +28,8 @@
             <public-packages>
                 <package>com.formdev.flatlaf</package>
```

Comment:
I'd rather go with:
```
<subpackages>com.formdev.flatlaf</subpackages>
```

Reply:
Thanks, good point I pushed another commit. I actually wasn't aware of this option as I'm more familiar with the Mavenized platform, which is a bit different on this matter.